ext/voice.py

```diff
@@ -0,0 +1,35 @@
+import discord
+from discord.ext import commands
+
+class Voice(commands.Cog):
+
+  def __init__(self, bot):
+    self.bot = bot
+    self.voice_roles = { # voice roles mapped by their channel
+      "Voice Channel 1": "Voice Chatter 1",
+      "Voice Channel 2": "Voice Chatter 2"
+    }
+    
+  @commands.Cog.listener()
+  async def on_voice_state_update(self, member, before, after):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [invalid syntax](https://app.codacy.com/app/RainbowLegend/aerobot/pullRequest?prid=4710794)

Reply:
Not a syntax error on py version 3, on earlier versions they do however but shouldn't be an issue on 3.6